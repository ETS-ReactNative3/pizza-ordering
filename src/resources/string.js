const string = {
  restaurantName: 'Pizza Pazzi',
};

export {string};
