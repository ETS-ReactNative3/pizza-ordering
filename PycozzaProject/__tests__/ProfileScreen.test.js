import React from 'react';
import { shallow, mount } from 'enzyme'
import ProfileScreen from '../app/views/Profile/Profile.screen';
import string from '../app/resources/strings';
import configureMockStore from "redux-mock-store";
import { Provider } from "react-redux";
const mockStore = configureMockStore();
const store = mockStore({});

describe('Profile screen Test', () => {
    it('should render snapshot test for profile screen', () => {
        const wrapper = mount(
            <Provider store={store}>
                <ProfileScreen />
            </Provider>
        )
        expect(wrapper).toMatchSnapshot();
    })
}
)