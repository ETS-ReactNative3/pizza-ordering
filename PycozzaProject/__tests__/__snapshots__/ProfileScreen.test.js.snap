// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile screen Test should render snapshot test for profile screen 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(ProfileScreen)>
    <ProfileScreen
      addUser={[Function]}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "marginTop": 20,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "marginTop": 20,
            }
          }
        >
          <OvalShape>
            <View
              style={
                Object {
                  "alignSelf": "center",
                  "backgroundColor": "#0b2031",
                  "borderRadius": 400,
                  "height": 1334,
                  "position": "absolute",
                  "top": -897,
                  "width": 1434,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignSelf": "center",
                    "backgroundColor": "#0b2031",
                    "borderRadius": 400,
                    "height": 1334,
                    "position": "absolute",
                    "top": -897,
                    "width": 1434,
                  }
                }
              />
            </View>
          </OvalShape>
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "marginTop": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flex": 1,
                  "marginTop": 20,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "flex": 5,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flex": 5,
                    }
                  }
                >
                  <Image
                    source={
                      Object {
                        "uri": "https://image.freepik.com/free-vector/profile-icon-male-avatar-hipster-man-wear-headphones_48369-8728.jpg",
                      }
                    }
                    style={
                      Object {
                        "borderRadius": 200,
                        "height": 200,
                        "resizeMode": "center",
                        "width": 200,
                      }
                    }
                  >
                    <Image
                      source={
                        Object {
                          "uri": "https://image.freepik.com/free-vector/profile-icon-male-avatar-hipster-man-wear-headphones_48369-8728.jpg",
                        }
                      }
                      style={
                        Object {
                          "borderRadius": 200,
                          "height": 200,
                          "resizeMode": "center",
                          "width": 200,
                        }
                      }
                    />
                  </Image>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "flexDirection": "row",
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flexDirection": "row",
                        }
                      }
                    >
                      <Text
                        style={
                          Object {
                            "fontSize": 30,
                            "fontWeight": "bold",
                          }
                        }
                      >
                        <Text
                          style={
                            Object {
                              "fontSize": 30,
                              "fontWeight": "bold",
                            }
                          }
                        >
                          You have not logged in
                        </Text>
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </ProfileScreen>
  </Connect(ProfileScreen)>
</Provider>
`;
